pkgs/top-level/python-packages.nix

```
@@ -8714,6 +8714,8 @@ in {
 
   python3-saml = callPackage ../development/python-modules/python3-saml { };
 
+  python3-xcaplib = callPackage ../development/python-modules/python3-xcaplib { };
```

```suggestion
  xcaplib = callPackage ../development/python-modules/xcaplib { };
```
This matches the import name and is unused